README.md

```diff
@@ -19,7 +19,7 @@ python3 -m pip install --upgrade clp-ffi-py
 Note:
 
 - Python 3.7 or higher is required.
-- Only Linux and macOS are supported at present.
+- Tested only on Linux, macOS and Windows.
```

Comment:
Do we still need work "only", lol

Reply:
lol i was thinking about Solaris and FreeBSD when i wrote this, but yeah the word seems a bit unnecessary here